cron.go

```diff
@@ -110,28 +111,28 @@ func parseCronField(fieldIdx int, fieldStr string) (field, error) {
 	fields := make([]field, 0, len(parts))
 	for _, p := range parts {
 		if p == "" {
-			return nil, errors.Errorf("could not parse the cron field: %s. invalid list item: %s", fieldStr, p)
+			return nil, fmt.Errorf("could not parse the cron field: %s. invalid list item: %s", fieldStr, p)
 		}
 		possibleRangeFields := strings.Split(p, "-")
 
 		if len(possibleRangeFields) == 1 {
 			parsed, err := parseValue(fieldIdx, possibleRangeFields[0])
 			if err != nil {
-				return nil, errors.Wrapf(err, "could not parse cron field: %s. invalid list item: %s", fieldStr, p)
+				return nil, fmt.Errorf("%w could not parse cron field: %s. invalid list item: %s", err, fieldStr, p)
```

Comment:
This should be
```
fmt.Errorf("could not parse cron field: %s. invalid list item %q: %w", fieldStr, p, err)
```
Original lib puts err to the end of the message, see https://go.dev/play/p/xr6ev6Q1Hyt